codegen-renderers/src/test/resources/java/ramlTestFiles/types/common.raml

```diff
@@ -0,0 +1,144 @@
+#%RAML 1.0 Library
```

Comment:
I like that you added these test files, but I would prefer to have a simpler example here.